bodhi/server/metadata.py

```diff
@@ -11,23 +11,21 @@
 # You should have received a copy of the GNU General Public License along
 # with this program; if not, write to the Free Software Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
-
-__version__ = '2.0'
-
-import os
 import logging
+import os
 import shutil
 import tempfile
 
-from urlgrabber.grabber import urlgrab
 from kitchen.text.converters import to_bytes
-
+from urlgrabber.grabber import urlgrab
 import createrepo_c as cr
 
+from bodhi.server.buildsys import get_session
 from bodhi.server.config import config
 from bodhi.server.models import Build, UpdateStatus, UpdateRequest, UpdateSuggestion
-from bodhi.server.buildsys import get_session
 
+
+__version__ = '2.0'
 log = logging.getLogger(__name__)
```

Comment:
I thought capitalized variables were global variables, not necessarily constant, I might be wrong though (and it doesn't really matter for the PR at hand)